content/transfers.html

```diff
@@ -35,13 +35,14 @@ <h3> Reason for Switching </h3>
 </span>
 The Engineering program has a relatively rigid course schedule which offers students the flexibility to choose courses usually only in their fourth year. 
 The Computer Science program, on the other hand, is less restrictive and allows students to customize their schedules freely. 
-<br /> <br />
-Other reasons for transferring included avoiding a course, graduating early, and accommodating for an exchange term. 
 </p>
 <div class="half second">
     <span class="graph-title">reasons for transferring</span>
     <div id="reasons-transferred"></div>
 </div>
+<p class="full">
+    Other reasons for transferring included avoiding a course, graduating early, and accommodating for an exchange term.
```

Comment:
-for

Reply:
Also, about CEGEP transfer credits, I'd like to get the most accurate information possible there, so that people don't get misinformation. That page that we link doesn't actually help. As we understand it, the deal is: after two years of CEGEP, exemptions from CEGEP Cal 2 + 3 = MATH 117; plus one other course (CHE 102 or a complementary studies elective). We have also allowed students to convert science electives to open electives but I don't want to put that on the webpage.